### ISSUE TITLE
[Embed] Accessibility issues flagged for Youtube

### ISSUE THREAD

**Issue by abhishekagg19** [adobe/aem-core-wcm-components#902]
Accessibility report is flagging following error for iframe used for Youtube embed:
This iframe element does not have a mechanism that allows an accessible name value to be calculated.

Embed Core component should be enhanced to address this accessibility issue:
Use the title attribute to define an accessible name for an iframe element.
Use the aria-label attribute to define an accessible name for an iframe element.

**Comment by richardhand** [comment]
Hi @abhishekagg19, thanks for the report. This makes sense for the youtube _embeddable_ that is provided out of the box. If I understand, the `aria-label` is enough.

The embed component also has an oEmbed url provider, unfortunately there we simply insert the markup returned by the oEmbed endpoint, so we have no control over accessibility.

**Comment by abhishekagg19** [comment]
@richardhand  Addition of aria-label for Youtube should be good enough.

Regarding oEmbed URL provider, accessibility related guideline can be added in documentation stating that onus is on the provider implementation.

**Comment by richardhand** [comment]
@abhishekagg19 - created a pull request for this at 915, could you take a look? Thanks!

**Comment by vladbailescu** [comment]
Fix merged to `master` in 4d1ad92
